src/storage-folder.coffee

```diff
@@ -0,0 +1,27 @@
+path = require "path"
+fs = require "fs-plus"
+
+module.exports =
+class StorageFolder
+  constructor: (containingPath) ->
+    @path = path.join(containingPath, "storage")
+
+  store: (name, object) ->
+    fs.writeFileSync(@pathForKey(name), JSON.stringify(object), 'utf8')
+
+  load: (name) ->
+    try
+      stateString = fs.readFileSync(@pathForKey(name), 'utf8')
```

Comment:
Should it be deleted after it is read? So if Atom is crashing because of the window state it doesn't keep retrying on subsequent launches.